### ISSUE TITLE
bitbucket_group issue.

### ISSUE THREAD

**Issue by raypettersen** [zahiar/terraform-provider-bitbucket#116]
Hi

When trying to create a group using bitbucket_group, the group is created but terraform returns an error that the group can't be found.

```
bitbucket_group.this: Creating...
╷
│ Error: unable to get group with error: no group found
│
│   with bitbucket_group.this,
│   on main.tf line 18, in resource "bitbucket_group" "this":
│   18: resource "bitbucket_group" "this" {
│╵
```

Code in question:

```
resource "bitbucket_group" "this" {
  workspace  = data.bitbucket_workspace.cillco.name
  name       = "ExampleGroup"
  auto_add   = false
  permission = "write"
}
```

Group can be found, so it def got created. Unsure if I am doing something wrong or if this is an actual bug?

**Comment by raypettersen** [comment]
Having a similar issue with the data function.

```
data "bitbucket_group" "Developers" {
  workspace = data.bitbucket_workspace.cillco.uuid
  slug      = "Developers"
}
```

And the group is certainly there. Perhaps Atlassian changed something on their side, resulting in issues with group lookup?

**Comment by zahiar** [comment]
@raypettersen I'm unable to reproduce this, feels like permission issues 🤔

**Comment by raypettersen** [comment]
That's odd. I created a new app password and granted all privileges that was possible during testing, and still the same problem. The provider manages to create groups, but seems unable to read them.

**Comment by zahiar** [comment]
I just saw the Group API has changed (only because the acceptance tests are failing). The `auto_add` is no longer there and instead there's a new permission `none`. Looks like the group is auto-added to all repositories with either none, read, write or admin permissions. And you can then override it on a per repository basis, if need be.

**Comment by raypettersen** [comment]
Good catch. 👍

**Comment by zahiar** [comment]
@raypettersen Just released version 1.3.0, after much flakiness from Bitbucket's API, anyway I'll close out this ticket and please raise a new one if you come across any issues 🙂
